profile/Bbmri-observation-bodyheight.StructureDefinition.json

```diff
@@ -0,0 +1,52 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "https://fhir.bbmri.de/StructureDefinition/Bbmri-observation-bodyheight",
+  "name": "Bbmri-observation-bodyheight",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Observation",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/bodyheight",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Observation.subject",
+        "path": "Observation.subject",
+        "type": [
+          {
+            "code": "Reference",
+            "targetProfile": [
+              "http://fhir.germanbiobanknode.de/fhir/StructureDefinition/GbaPatient"
+            ]
+          }
+        ]
+      },
+      {
+        "id": "Observation.effective[x]",
+        "path": "Observation.effective[x]",
+        "type": [
+          {
+            "code": "dateTime"
+          }
+        ]
+      },
+      {
+        "id": "Observation.value[x]:no name",
+        "path": "Observation.value[x]",
+        "sliceName": "no name"
```

Comment:
what is that?

Reply:
A Forge artifact inhereted from the FHIR core profiles. I removed it from our inherited profile, so as soon the issue is fixed in Forge / the core profiles, our profiles should also be displayed correctly.